assets/scss/app.scss

```diff
@@ -28,6 +28,29 @@
   width: 100%;
 }
 
+header {
+  margin-top: 1.25em;
+  margin-bottom: 0.5em;
+}
```

Comment:
see below comment

Reply:
I had a think.  The header is a very important element used quite frequently.  This will impact every page.  Can we discuss this? :)